pkg/kubelet/rkt/rkt.go

```diff
@@ -1069,10 +1073,16 @@ func (r *Runtime) Type() string {
 }
 
 func (r *Runtime) Version() (kubecontainer.Version, error) {
+	if err := r.getVersions(); err != nil {
+		return nil, err
+	}
 	return r.binVersion, nil
```

Comment:
@Random-Liu Hmm, getVersions() updates `r.systemdVersion` `r.binVersion` `r.appcVersion` `r.apiVersion`

But I noticed now they have a write-write race now instead of read-write race :( Though either case doesn't really matter.

Will try to update again. Thanks.